pkg/api/ping/pinghandler.go

```diff
@@ -23,7 +23,7 @@ func Handler() http.Handler {
 
 		request.RequestID = req.Header.Get("Request-Id")
 
-		response := PingResponse{Status: true, Data: "pong"}
+		response := Response{Status: true, Data: "pong"}
```

Comment:
ideally adding version is helpful in ping, and more things like `db.ping` and posting status helps than simple pong, we could do this later as well